client/me/help/help-contact/index.jsx

```diff
@@ -260,7 +260,14 @@ module.exports = React.createClass( {
 		}
 
 		if ( ! ( olark.isOlarkReady && sitesInitialized ) ) {
-			return <div className="help-contact__placeholder" />;
+			return (
+				<div className="help-contact__placeholder">
+					<HelpContactForm disabled={true}
+						showSubjectField={ false }
+						onSubmit={ function() {} }
+						buttonLabel={ this.translate( 'Contact us' ) }/>
+				</div>
+			);
 		}
 
 		if ( olark.details.isConversing && olark.isOperatorAvailable ) {
```

Comment:
I suggested this because I saw it done in `HelpSearch`. If it seems hacky then maybe it is.